plugins/flashrom/fu-reset-cmos.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2021 Sean Rhodes <sean@starlabs.systems>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+#include "config.h"
+
+#include <libflashrom.h>
+#include <sys/io.h>
+#include <unistd.h>
+
+#include "fu-reset-cmos.h"
+#include "glib.h"
```

Comment:
since you're including <fwupdplugin.h> in fu-reset-cmos.h, I don't think you need a glib include anymore